packages/twenty-server/src/database/typeorm-seeds/workspace/opportunity.ts

```diff
@@ -31,7 +31,7 @@ export const seedOpportunity = async (
         amountCurrencyCode: 'USD',
         closeDate: new Date(),
         probability: 0.5,
-        stage: 'new',
+        stage: 'NEW',
```

Comment:
@thomtrp Did you create a command to migrate this ? Or it will be handled manually ?

Reply:
Good question, I did not create a command yet. It may be the best thing to do. Checking with you separately!